newm-server/src/main/kotlin/io/newm/server/features/minting/repo/MintingRepositoryImpl.kt

```diff
@@ -85,13 +103,16 @@ class MintingRepositoryImpl(
             configRepository.getString(CONFIG_KEY_MINT_STARTER_TOKEN_UTXO_REFERENCE).toReferenceUtxo()
         val scriptUtxoReference = configRepository.getString(CONFIG_KEY_MINT_SCRIPT_UTXO_REFERENCE).toReferenceUtxo()
 
-        val signingKeys = listOf(cashRegisterKey, paymentKey, collateralKey)
+        val signingKeys = listOfNotNull(cashRegisterKey, moneyBoxKey, paymentKey, collateralKey)
 
         var transactionBuilderResponse =
             buildMintingTransaction(
                 paymentUtxo,
                 cashRegisterUtxos,
-                cashRegisterKey.address,
+                changeAddress = cashRegisterKey.address,
```

Comment:
nit: would be good to add param names for all arguments of `buildMintingTransaction` here

Reply:
done